src/lambda/models/samLambdaRuntime.ts

```diff
@@ -34,6 +34,20 @@ export const nodeJsRuntimes: ImmutableSet<Runtime> = ImmutableSet<Runtime>([
     'nodejs14.x',
     'nodejs12.x',
 ])
+export function getNodeMajorVersion(version?: string): number | undefined {
+    if (!version) {
+        return undefined
+    }
+
+    const match = version.match(/^nodejs(\d+)\./)
```

Comment:
oh, we probably want this to be case insensitive. OTOH, the test will catch it, so I guess doesn't matter.
```suggestion
    const match = version.match(/^nodejs(\d+)\./i)
```